etc/kayobe/stackhpc-monitoring.yml

```diff
@@ -1,5 +1,9 @@
 ---
 # StackHPC monitoring configuration
+###############################################################################
+# Prometheus server configuration
+
+stackhpc_prometheus_query_lookback_delta: "{{ [redfish_exporter_scrape_interval, stackhpc_os_capacity_scrape_interval, 300] | max }}s"
```

Comment:
Do you need to add a few seconds here to avoid some of the edge effects you saw in testing?

Reply:
I am also wonder if we need to change the openstack exporter threshold to be included in here as well. For larger clouds, its probably a bit too aggressive by default.

Reply:
I've added 30 seconds to give some leeway. Have also added an openstack exporter scrape interval variable and added it the max calculation.